acceptance/tests/direct_puppet/static_catalog_env_control.rb

```diff
@@ -0,0 +1,348 @@
+test_name "Environment control of static catalogs" 
+require 'json'
+
+@testroot = master.tmpdir(File.basename(__FILE__, '/*'))
+### @testroot = "/etc/puppetlabs"
+
+@coderoot = "#{@testroot}/code"
+
+@master_opts = {
```

Comment:
This is unnecessary given the puppetserver config below. Should we skip this test if not using puppetserver?